script/DeployGMDMarket.s.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Script.sol";
+import "forge-std/StdJson.sol";
+
+import { BaseScript } from "./BaseScript.sol";
+import { BaseDeployScript } from "./BaseDeployScript.sol";
+import { GMDYieldSource } from "../src/sources/GMDYieldSource.sol";
+
+contract DeployGMDMarket is BaseScript, BaseDeployScript {
+    using stdJson for string;
+
+    function setUp() public {
+        init();
+    }
+
+    function run() public {
+        vm.startBroadcast(pk);
+        runDeploy(DeployOptions({
+            yieldSource: new GMDYieldSource(),
+            slug: "gmd",
+            discountDaily: 158e10, // As of 6/5/22
```

Comment:
this number is wrong so just put in 1e15 and remove the date comment. we can fill it in later